album.go

```diff
@@ -0,0 +1,16 @@
+package spotify
+
+type Album struct {
+	Meta
+	AlbumType        AlbumType `json:"album_type"`
+	AvailableMarkets []string  `json:"available_markets"`
+	Images           []Image
+	Name             string
```

Comment:
It would be great to include JSON tags for these too

Reply:
They're not super necessary since the default JSON implementation handles this correctly (it will try and dow-ncase the field name during unmarshalling for a match). Happy to add them anyway!